packages/itwinui-react/src/core/Toast/Toast.tsx

```diff
@@ -169,27 +169,37 @@ export const Toast = (props: ToastProps) => {
     return { translateX, translateY };
   };
 
+  const motionOk = getWindow()?.matchMedia(
+    '(prefers-reduced-motion: no-preference)',
+  )?.matches;
```

Comment:
`useMediaQuery` would be better here. it is more optimized.

```jsx
  const motionOk = useMediaQuery('(prefers-reduced-motion: no-preference)');
```

Reply:
This has been changed